modules/genomic_browser/php/view_genomic_file.class.inc

```diff
@@ -35,13 +35,12 @@ class View_Genomic_File extends \NDB_Form
     /**
     * Determine whether the user has permission to view this page
     *
+    * @param \User $user The user whose access is being checked
+    *
     * @return bool whether the user has access
     */
-    function _hasAccess()
+    function _hasAccess(\User $user) : bool
     {
-        $user =& \User::singleton();
-
-        // Add site control? See Imaging Browser example
```

Comment:
would we maybe need to keep this comment?

Reply:
done